static/cfg-view.js

```diff
@@ -117,8 +117,8 @@ define(function(require){
             valueField: 'name',
             labelField: 'name',
             searchField: ['name'],
-            options: this.fnNames.length ? this.adaptStructure(this.fnNames) : [{name:"no functions"}],
-            items: this.fnNames.length ? [this.currentFunc] : [{name:"no functions"}]
+            options: this.fnNames.length ? this.adaptStructure(this.fnNames) : [{name:"the input contain no function"}],
```

Comment:
Capitalize the sentence. Also, you have a grammar mistake. A fixed version coud say `The input contains no functions`